src/sentry/web/frontend/group_tag_export.py

```diff
@@ -67,19 +69,15 @@ def get(self, request, organization, project, team, group_id, key):
         except Group.DoesNotExist:
             raise Http404
 
-        if TagKey.is_reserved_key(key):
+        if tagstore.is_reserved_key(key):
             lookup_key = 'sentry:{0}'.format(key)
         else:
             lookup_key = key
 
         # validate existance as it may be deleted
         try:
-            TagKey.objects.get(
-                project_id=group.project_id,
-                key=lookup_key,
-                status=TagKeyStatus.VISIBLE,
-            )
-        except TagKey.DoesNotExist:
+            tagstore.get_tag_key(group.project_id, lookup_key)
+        except ObjectDoesNotExist:
```

Comment:
This is an implementation detail leaking out. I think `tagstore` should handle this internally and raise it's own `TagKeyNotFound` or something that makes sense as an abstraction. The fact that you've had to drop down to `ObjectDoesNotExist` is the red flag to me since you already now don't know what the model is, etc.

Reply:
Damn, you caught me. I actually realized that too but forgot to switch it over.